x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_creation/select_engine_type.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiButton,
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiSpacer,
+  EuiStepsHorizontal,
+  EuiText,
+  EuiTextAlign,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  ENGINE_CREATION_SELECT_APP_SEARCH_TITLE,
+  ENGINE_CREATION_SELECT_APP_SEARCH_DESCRIPTION,
+  ENGINE_CREATION_SELECT_ELASTICSEARCH_TITLE,
+  ENGINE_CREATION_SELECT_ELASTICSEARCH_DESCRIPTION,
+  ENGINE_CREATION_NEXT_STEP_BUTTON_LABEL,
+} from './constants';
+
+import { EngineCreationLogic, EngineCreationSteps } from './engine_creation_logic';
+
+export const SelectEngineType: React.FC = () => {
+  const { engineType } = useValues(EngineCreationLogic);
+  const { setEngineType, setCreationStep } = useActions(EngineCreationLogic);
+
+  return (
+    <div className="entSearch__createEngineLayout">
+      <EuiStepsHorizontal
+        steps={[
+          {
+            title: 'Search engine type',
+            status: 'current',
+            onClick: () => {},
+          },
+          {
+            title: 'Configuration',
+            onClick: () => {
+              setCreationStep(EngineCreationSteps.ConfigureStep);
+            },
+          },
+          {
+            title: 'Review',
+            status: 'disabled',
+            onClick: () => {},
+          },
+        ]}
+      />
+
+      <EuiSpacer />
+
+      <EuiPanel hasBorder paddingSize="l">
+        <EuiTextAlign textAlign="center">
+          <EuiTitle>
+            <h2>
+              {i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engineCreation.SelectEngineTypeForm.title',
+                {
+                  defaultMessage: 'Select a search engine type',
+                }
+              )}
+            </h2>
+          </EuiTitle>
+        </EuiTextAlign>
+
+        <EuiSpacer />
+
+        <EuiText color="subdued" textAlign="center">
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engineCreation.SelectEngineTypeForm.description',
+            {
+              defaultMessage: `
+                You can now create search engines that are backed by an
+                Elasticsearch index. We need some good copy here explaining what
+                all the hubub about this new method is and a link to learn more
+                about it.
+              `,
+            }
+          )}
+        </EuiText>
+
+        <EuiSpacer />
+        <EuiSpacer />
```

Comment:
You can pass `size` prop instead of using this twice

Reply:
It turns out two are necessary. Something to do with margins. @daveyholler understands this better than I do.